test/tools/file-stasher.js

```diff
@@ -110,7 +110,7 @@ describe('file upload', () => {
       .catch(done);
   });
 
-  it('should fail if not being called from an hydrator event', (done) => {
+  it('should fail if being called from a trigger event', (done) => {
```

Comment:
Looks good! Please consider adding a test for a search (and maybe passing ones for hydration and create) since you're being specific here.

Reply:
All set!